### PR TITLE
Replaced disable with deactivate for latest docs

### LINKED ISSUE #303: Update docs to use deactivate instead of disable
The Harbor interface has been updated to use more inclusive terminology in https://github.com/goharbor/harbor/issues/16248. We should update the docs to reflect the changes in the interface.

This will include updating wording through the docs and screenshots. 

https://github.com/goharbor/harbor/issues/16248 updated the word `disable` to `deactivate`. So this is the change that needs to be made.

Hopefully we can do this using find replace on the website content in an editor.

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
Fixes #303.

### CODE REVIEW

**Review by Vad1mo — APPROVED** [review]

File: docs/administration/backup-restore/_index.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ If you have any issue while backing up or restoring, please refer to Velero's [t
 
 ## Limitations
 * **The upload purging process may cause backup failure**  
-  A purging process starts in the `registry` pod by default, it removes the unused files under the upload directory periodically and cannot be disabled without restarting. This may impact the backup when using Restic and cause failures.  
+  A purging process starts in the `registry` pod by default, it removes the unused files under the upload directory periodically and cannot be deactivated without restarting. This may impact the backup when using Restic and cause failures.
```
[the remark this answers is not in the document]
Comment (PaarthAgarwal) [reply docs/administration/backup-restore/_index.md:89]: I checked the deployment preview. It doesn't apply to docs version 2.7 and below

[the remark this answers is not in the document]
Comment (OrlinVasilev) [reply docs/administration/backup-restore/_index.md:89]: that's needs to be cherrypicked into release-2.7 as well

**Comment by Vad1mo** [comment]
@PaarthAgarwal there appears to be a merge conflict, could you help resolve that. 

It is on the finish line!

**Comment by OrlinVasilev** [comment]
@PaarthAgarwal can you update the branch and fix conflicts so we can merge that , thanks mate :)

**Comment by OrlinVasilev** [comment]
@PaarthAgarwal are you still up for that?

**Pull request merged**